releases/release-1.30/README.md

```diff
@@ -0,0 +1,120 @@
+| 1.30.0-rc.0 released                                                                   | Branch Manager                | Tuesday 26th March 2024                                                                                            | week 12  | [1.30-blocking], [master-blocking], [master-informing] |
+| Docs complete — All PRs reviewed and ready to merge                                    | Docs Lead                     | Tuesday 26th March 2024                                                                                            | week 12  |                                                        |
+| Major Themes complete                                                                  | Release Notes Lead            | Tuesday 2nd April 2024                                                                                            | week 13  |                                                        |
+| 1.30.0-rc.1 released                                                                   | Branch Manager                | Tuesday 2nd April 2024                                                                                            | week 13  |                                                        |
+| 1.30.0-rc.2 released                                                                   | Branch Manager                | Thursday 11th April 2024                                                                                            | week 14  | [1.30-blocking], [master-blocking], [master-informing] |
+| Release Notes complete — reviewed & merged to https://github.com/kubernetes/kubernetes | Release Notes Lead            | Wednesday 13th December 2024                                                                                          | week 15  |                                                        |
+| **v1.30.0 released**                                                                   | Branch Manager                | Wednesday 17th April 2024                                                                                          | week 15  |                                                        |
+| Release blog published                                                                 | Comms                         | Wednesday 17th April 2024                                                                                          | week 15  |                                                        |
+| **[Thaw]**                                                                             | Branch Manager                | Wednesday 17th April 2024                                                                                          | week 15  |                                                        |
+
+## Phases
+
+Please refer to the [release phases document](../release_phases.md).
+
+[k8s130-calendar]: https://bit.ly/k8s-release-cal
+[Internal Contact Info]: https://bit.ly/k8s130-contacts
```

Comment:
fyi I didn't do this for the v1.28 release because it didn't feel necessary but Priyanka did

Reply:
Yeah fair enough, it's redundant and it makes the doc super long. Removed